src/qt/editor/inviwomainwindow.cpp

```diff
@@ -576,8 +591,11 @@ void InviwoMainWindow::addActions() {
                 auto path = QString::fromStdString(moduleWorkspacePath + "/" + item);
                 connect(action, &QAction::triggered, this, [this, path]() {
                     // open as regular workspace with proper filename if control is pressed
-                    bool controlPressed = (app_->keyboardModifiers() == Qt::ControlModifier);
-                    if (askToSaveWorkspaceChanges()) {
+                    if (app_->keyboardModifiers() == Qt::AltModifier) {
```

Comment:
Ctrl is commonly used to append/copy, so we should use that instead of Alt in my opinion.

We could use Alt to open example/test workspaces as "editable"

Reply:
We have used Ctrl for that before and I did not want to change that... I think either is quite arbitrary.

What would be nice would be to change the text in the menu when holding down the buttons to indicate that the behavior changes, But I don't know how...

Reply:
quick and dirty. Might need some fine tuning...
```c++
namespace detail {

class MenuChangeEventFilter : public QObject {
public:
    MenuChangeEventFilter(QObject* parent, QAction* action = nullptr, const QString& activeTxt = "",
                          const QString& inactiveTxt = "")
        : QObject(parent), action_(action), active_(activeTxt), inactive_(inactiveTxt) {}
    virtual bool eventFilter(QObject*, QEvent* ev) override {
        if (action_) {
            if (ev->type() == QEvent::KeyPress) {
                if (QGuiApplication::keyboardModifiers().testFlag(Qt::ControlModifier)) {
                    action_->setText(active_);
                } else {
                    action_->setText(inactive_);
                }
                return true;
            }
        }
        return false;
    }

private:
    QAction* action_;
    QString active_;
    QString inactive_;
};

...
        auto reloadStyle = fileMenuItem->addAction("Reload Style sheet");
        installEventFilter(new detail::MenuChangeEventFilter(
            this, reloadStyle, "Append Style sheet", "Reload Style sheet"));


}  // namespace detail

Reply:
Was also considering if an event filter might work... Any idea what event that will end up in the eventFilter? All event ever or only then ones generated when the menu is open? or something in-between?

Reply:
Append is also considered for drag and drop? Then ctrl is quite natural I think

Reply:
In this case the event filter is installed on the entire application window to capture changes in the modifier (key event). Could not get it to work on QAction level. Probably the filter should not accept/consume the event either.

Reply:
Example above is now more reliable, but the event gets triggered quite often. Installing the event filter onto the core application is necessary to catch modifier key releases (see [Stackoverflow](https://stackoverflow.com/a/11026014)).